pyface/image/tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
```

Comment:
we're leaving the `__init__` files empty and we're not adding the copyright headers in them.

```suggestion
```